### PR TITLE
Added example for auth with ENV vars

### PR DESCRIPTION
**Pull request by alexfilatov**
Please consider adding this example to Readme as this shows simple way how to authenticate with ENV variables "GOOGLE_CLIENT_ID", "GOOGLE_CLIENT_EMAIL", "GOOGLE_ACCOUNT_TYPE", "GOOGLE_PRIVATE_KEY"

Thank you!

### CODE REVIEW

**Comment by alexfilatov** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -152,6 +152,21 @@ export GOOGLE_CLIENT_EMAIL=xxxx@xxxx.iam.gserviceaccount.com
...
 
+```ruby
+require 'googleauth'
+require 'google/apis/drive_v3'
+
+Drive = ::Google::Apis::DriveV3
+drive = Drive::DriveService.new
+
+# Auths with ENV vars "GOOGLE_CLIENT_ID", "GOOGLE_CLIENT_EMAIL", "GOOGLE_ACCOUNT_TYPE", "GOOGLE_PRIVATE_KEY"
+auth = ::Google::Auth::ServiceAccountCredentials.make_creds(scope: 'https://www.googleapis.com/auth/drive')
```
Comment (TheRoyalTnetennba) [inline README.md]: Can you make these lines narrower?

**Comment by andrewhaines** [comment]
@TheRoyalTnetennba Do you still need someone to finish this up? It also seems that adding the `json_key_io` parameter is required even if you set the `ENV` vars.

**Comment by sduskis** [comment]
@TheRoyalTnetennba, what's the state of this PR?

**Comment by alexfilatov** [comment]
sorry for delay, made lines more narrow as requested.

File: README.md (hunk 1 above)
Comment (alexfilatov) [reply README.md]: done @TheRoyalTnetennba

**Pull request merged**